packages/next/build/webpack-config.ts

```diff
@@ -86,6 +86,8 @@ function getOptimizedAliases(isServer: boolean): { [pkg: string]: string } {
     'object.assign$': path.join(shimAssign, 'index.js'),
     'object.assign/polyfill': path.join(shimAssign, 'polyfill.js'),
     'object.assign/shim': path.join(shimAssign, 'shim.js'),
+
+    url: 'native-url',
```

Comment:
For consistency, let's do:
```suggestion
    // Replace: full URL polyfill with platform-based polyfill
    url: require.resolve('native-url'),
```